app/config.py

```diff
@@ -17,14 +18,13 @@ class Settings(BaseSettings):
     fertiscan_db_url: PostgresDsn
     fertiscan_schema: str
     fertiscan_storage_url: str
-    frontend_url: str = "*"
     openai_api_deployment: str = Field(alias="azure_openai_deployment")
     openai_api_endpoint: str = Field(alias="azure_openai_endpoint")
     openai_api_key: str = Field(alias="azure_openai_key")
     otel_exporter_otlp_endpoint: str | None = None
     swagger_path: str = "/docs"
     upload_folder: str = "uploads"
-    testing: str = "false"
+    allowed_origins: list[str] = Field(["*"], alias="frontend_url")
 
 
```

Comment:
We shouldn't allow star, specify a list of URLs. ["http://localhost:3000", "https://fertiscan.inspection.alpha.canada.ca"]
or use env vars this is how I did it in Nachet 
```Python
cors_settings = {
    "allow_origin": [URL_LOCAL] if ENVIRONMENT == "local" else [URL_PROD],
    "allow_methods": ["GET", "POST", "OPTIONS"],
    "allow_credentials": True,
    "max_age": 86400
}

app = Quart(__name__)
app = cors(app, **cors_settings)
```

If you want to read cookies from the user, the browser will error out if you specify a * allowed origin on the server. I'm surprised this config did not error out for you.
. * also allows XSS attacks